Gaming The Syllabus/Assets/Scripts/Combat/UI/CombatStarter.cs

```diff
@@ -25,4 +25,12 @@ public void StartCombat()
 
         CombatManager.StartCombat(enemies: combatUnits.ToArray());
     }
+
+    void Update()
+    {
+        if (CombatManager.CheckCombatIsOver() && CombatManager.player.IsAlive)
```

Comment:
We could consider an alternate more event-driven approach. CombatManager could remember what the trigger for this combat was and then set do this logic when combat is finished so we aren't polling every frame. That said it doesn't really matter and might be more trouble than it's worth so... this seems good to me